baseline/pytorch/classify/train.py

```diff
@@ -46,14 +51,14 @@ def __init__(self, model, **kwargs):
         self.optimizer = OptimizerManager(model, **kwargs)
         self.model = model
         if self.gpus > 0 and self.model.gpu:
-            self.crit = model.create_loss().cuda()
+            self.crit = model.create_loss(**kwargs).cuda()
```

Comment:
need to check if the parameters exist for this function before we jam kwargs in there -- if its a legacy addon, it might not have these args and then it will fail to run.

Reply:
check signature before passing args.  this could get us in trouble if somebody forgets it, but it will at least allow legacy addons to work as before